### DISCUSSION TITLE
pinia 2.0.19+ incompatible with older browsers

### DISCUSSION THREAD

**Discussion by samardzicneo** [vuejs/pinia#1626] [Help and Questions]
### Reproduction

\-

### Steps to reproduce the bug

1. Install pinia 2.0.19 or later in a Vue 2 project and use it somewhere
2. Build the project
3. Try to open it on Chrome 34 or a similar old browser

### Expected behavior

Everything should work as it would with older versions of pinia.

### Actual behavior

There is a usage of "const" in the source which means that the browser will refuse to read the entire bundle -> broken application.

### Additional information

Updating to a newer browser is **not possible** because Chrome 34 is the only browser available on older LG TVs, which we are developing for.

**Comment by posva** [comment]
An error stack trace would be helpful to find where the error comes from. Most of the time, it's a syntax error, so it should be easier to create a PR for this

**Reply by samardzicneo** [reply]
There is no stack trace, just a `Uncaught SyntaxError: Use of const in strict mode.` message colored red, no info on where it's coming from. The only place in our bundle where const is used is inside pinia though. The code looks like it has something to do with Devtools.

**Reply by posva** [reply]
hmm, can't go much further without that... Maybe try enabling source maps to see where it comes from.

The devtools code should be stripped in production. Are you by any chance not minifying your code?

**Reply by samardzicneo** [reply]
We have source maps enabled but I'm assuming that they're not read by the browser, because no stack trace shows up in the console. Might be a quirk of old Chromium, or a quirk of the 2014-era TVs.

We are using vue-cli with `NODE_ENV=production`. Afaik this means that it should be minified and tree-shaked.
